Pipfile

```diff
@@ -4,6 +4,7 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [packages]
+# Mysociety Django project
```

Comment:
Stylistic thing, but it’s always `mySociety`, rather than `Mysociety`, even at the start of a sentence.

Reply:
My apologies. Will fix. :)